src/dlgRoomExits.cpp

```diff
@@ -1044,9 +1044,24 @@ QAction* dlgRoomExits::getActionOnExit(QLineEdit* pExitLineEdit) const
     return nullptr;
 }
 
-/* static */ QString dlgRoomExits::generateToolTip(const QString& exitRoomName, const QString& exitAreaName, const bool outOfAreaExit, const int exitRoomWeight)
+/* static */ QString dlgRoomExits::generateToolTip(const QString& exitRoomName, const QString& exitAreaName, const bool exitRoomLocked, const bool outOfAreaExit, const int exitRoomWeight)
 {
     if (exitRoomName.trimmed().length()) {
+        if (exitRoomLocked) {
+            if (outOfAreaExit) {
+                return doubleParagraph.arg(tr("Exit to \"%1\" in area: \"%2\".")
```

Comment:
Can you say what kind of an exit it is? For examle "East exit to ..."

This will help clarity in the dialog to know what kind of an exit it is, since you can't immediately tell this from the dialog, for example we have no idea where any of these rooms are going:

![image](https://user-images.githubusercontent.com/110988/167785563-4b7b8143-a63a-4245-b382-5acaf9c78d36.png)

Reply:
Don't forget "tool-tip-text" generator is used for all the normal and the special exit roomID number display widgets!

It would be tricky (I think) to include the directional detail in the tool-tip and yet avoid assembling sentences for what will, don't forget, be translated, from "fragments" of text. It would be simpler if an indication of the direction, beyond the place-holder texts on non-existent exits, is really wanted to change the `QFrame`s around the controls for each normal exit direction to `QGroupBox`es (without the `QCheckBox` detail) but that would make the overall dialogue (a bit) larger...

Reply:
Yeah, and the relative placement is not the most intuitive thing that jumps out at you. Without context here, we really have no idea!

Reply:
I've looked and because i refactored the code a while back to use the same tool-tip text generator for all the places where a destination room id is entered, means it is not simple to introduce a *translatable* text fragment into the tool tips for normal exit direction.

I've looked and TBH it is much simpler to just switch to `QGroupBox`s - so that is what I am doing...

Reply:
... in the separate PR #6104.